cmd/autoscaler/main_test.go

```diff
@@ -19,6 +19,7 @@ import (
 	"fmt"
 	"strings"
 	"testing"
+	time "time"
```

Comment:
Just "time" it's simpler.

Reply:
I've no idea how that wound up being repeated, being repeated. I'll take it out.